mdata/cache/accnt/if.go

```diff
@@ -20,3 +20,8 @@ type EvictTarget struct {
 	Metric string
 	Ts     uint32
 }
+
+func NewAccnt(maxSize uint64) Accnt {
+	// currently we only know NewFlatAccnt
+	return NewFlatAccnt(maxSize)
+}
```

Comment:
> just completely drop it and directly call NewFlatAccnt() from the callsite of NewAccnt()

this.

same with the other wrapper, can be removed altogether and the concrete function called directly